File: aoj/volume100/nodejs/A10009.js
function main(lines) {
  var r = parseFloat(lines[0]);

  var area      = Math.PI * Math.pow(r, 2);
  var perimeter = Math.PI * 2 * r;
  console.log([
    area.toFixed(6),
    perimeter.toFixed(6),
  ].join(' '));
}

var chunks = [];
process.stdin.on('data', function(chunk) {
  chunks.push(chunk)
});
process.stdin.on('end', function() {
  var lines = chunks.join('').split('\n');
  main(lines);
});

File: aoj/volume100/scala/A10009.scala
object Main {
  def main(args: Array[String]) {
    val r = scala.io.StdIn.readDouble
    val area          = scala.math.Pi * scala.math.pow(r, 2)
    val circumference = 2 * scala.math.Pi * r
    printf("%.6f %.6f\n", area, circumference)
  }
}
